src/pages/index.js

```diff
@@ -199,9 +198,10 @@ function handleDeleteSubmit(evt) {
     closeModal(deleteModal);
   }
 }
-
+openModal;
 function openModal(modal) {
   modal.classList.add("modal_opened");
+  document.addEventListener("keydown", handleModalEvents);
 }
 
 function closeModal(modal) {
```

Comment:
* The checklist says: an event listener for closing the popup by pressing Esc is added when the popup is opened, and is removed when it's closed.

* Please, check your project 6. You had this `Escape` handling there https://snipboard.io/b40skP.jpg  . You shouldn't have changed it because you need to add and remove the `Escape` listener correctly, according to the task and checklist 
* And the `overlay` handler was correct in your project 6. Please, have a look at your code there https://snipboard.io/dJAVWa.jpg